tsconfig.json

```diff
@@ -1,15 +1,16 @@
 {
-	"compilerOptions": {
-		"module": "commonjs",
-		"declaration": true,
-		"noImplicitAny": true,
-		"removeComments": true,
-		"preserveConstEnums": true,
-		"strictNullChecks": true,
-		"sourceMap": true,
-		"target": "es6",
-		"types": [ "bluebird-global" ],
-		"noUnusedParameters": true,
-		"noUnusedLocals": true
-	}
+    "compilerOptions": {
+        "module": "commonjs",
+        "declaration": true,
+        "noImplicitAny": true,
+        "removeComments": true,
+        "preserveConstEnums": true,
+        "strictNullChecks": true,
+        "sourceMap": true,
+        "target": "es6",
+        "baseUrl": "./typings",
```

Comment:
I'm surprised by `baseUrl`. I've never seen it before, and [the docs](https://www.typescriptlang.org/docs/handbook/module-resolution.html#base-url) say it exists mainly for AMD projects.

Is this so that `require('temp')` finds `typings/temp.d.ts`? In that case, I think you just need to ensure that the file is already included in the build process - i.e. add `typings/*.d.ts` to `includes` here or your `gulp.files()` argument above. The module shouldn't need to resolved correctly from its name because it's an ambient module declaration - it will define the module name you can load it by explicitly by itself, independent of its location, as long as it's in the build.